pkg/registration/hub/manager.go

```diff
@@ -64,10 +68,14 @@ func NewHubManagerOptions() *HubManagerOptions {
 func (m *HubManagerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringSliceVar(&m.ClusterAutoApprovalUsers, "cluster-auto-approval-users", m.ClusterAutoApprovalUsers,
 		"A bootstrap user list whose cluster registration requests can be automatically approved.")
+	fs.StringSliceVar(&m.EnabledRegistrationDrivers, "enabled-registration-drivers", m.EnabledRegistrationDrivers,
+		"A list of registration drivers enabled on hub.")
```

Comment:
this csr driver should be set by default, for backward compatible

Reply:
Addressed

Reply:
let's set default value in registration controller instead setting in the operator, so if the flag is not specified. we will still use the csr driver.

Reply:
Addressed